sites/broadbandtechreport/config/site.js

```diff
@@ -31,12 +31,14 @@ module.exports = {
     { href: '/business-services', label: 'Business Services' },
     { href: '/back-office', label: 'Back Office' },
     { href: '/metrics', label: 'Metrics' },
-    { href: '/diamond-reviews', label: 'Diamond Reviews' },
   ],
   secondaryNavItems: [
     { href: '/subscribe', label: 'Subscribe' },
-    // { href: '/magazine', label: 'Magazine' },
```

Comment:
@brandonbk This was removed intentionally since they do not have magazines. Sure you want it back in there?

Reply:
Odd...Deb included it in her full list of nav updates; likely a mistake.  I will remove it.